src/config.json

```diff
@@ -52,6 +52,20 @@
 			"zh": "继续阅读",
 			"es": "Seguir leyendo",
 			"ru": "Продолжить чтение"
+		},
+		"tutorial": {
+			"previous": {
```

Comment:
I think we can just reuse the existing `previous` & `next` -- I get (in English, anyways) it's a slight change but that should be fine and I think still makes sense?

Reply:
@rschristian Great feedback, i agree! In that case, i'll reuse the existing two properties and only add `help` property under the `tutorial` context.

Is it okay to do it this way? `help` may be used on other pages, placing it within the `tutorial` contexts seems better as their purposes differ.

Reply:
> Is it okay to do it this way? help may be used on other pages, placing it within the tutorial contexts seems better as their purposes differ.

I think that makes a lot sense, sounds great to me!